lib/chef/policy_builder/policyfile.rb

```diff
@@ -51,7 +51,17 @@ class UnsupportedFeature < StandardError; end
 
       class PolicyfileError < StandardError; end
 
-      RunListExpansionIsh = Struct.new(:recipes, :roles)
+      RunListExpansionIsh = Struct.new(:recipes, :roles) do
+        def to_hash_for_json
```

Comment:
Sorry I didn't think of this before, but it's probably best to name this method `to_hash` to keep compatible with `RunListExpansion`